app/ui.js

```diff
@@ -1089,7 +1084,7 @@ var UI = {
     },
 
     disconnectFinished: function (e) {
-        let wasConnected = UI.connected;
+        const wasConnected = UI.connected;
```

Comment:
Unnecessary change.

Reply:
const would be better, then you are using ```const ``` you tell other developers what it is an immutable entity.